NTupleAnalyzer_nanoAOD.py

```diff
@@ -159,6 +159,14 @@
 #_kinematicvariables += ['passWptCut','passZptCut','WorZSystemPt']
 #_kinematicvariables += ['WSystemPt','ZSystemPt']
 _kinematicvariables += ['matchedLQ']
+_kinematicvariables += ['recoSFMu1','recoSFupMu1','recoSFdownMu1']
```

Comment:
Please change naming convention to the following, e.g. this line would be

_kinematicvariables += ['mu1recoSF','mu1recoSFup','mu1recoSFdown']

etc. for mu2 and other SF